inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/SourceSnapshotOperation.java

```diff
@@ -104,13 +163,33 @@ public void close() {
         this.isClose = true;
     }
 
+    /**
+     * Get all tasks in a temporary state, and set up an ip-id-status map,
+     * the temporary state is greater than or equal to 200.
+     *
+     * @see org.apache.inlong.manager.common.enums.SourceState
+     */
+    private ConcurrentHashMap<String, ConcurrentHashMap<Integer, Integer>> getTaskIpAndStatusMap() {
+        ConcurrentHashMap<String, ConcurrentHashMap<Integer, Integer>> ipTaskMap = new ConcurrentHashMap<>();
```

Comment:
better to have default capacity

Reply:
I'll add a default capacity in the next PR.